tajo-core/src/test/java/org/apache/tajo/cli/tools/TestTajoDump.java

```diff
@@ -64,4 +64,15 @@ public void testDump2() throws Exception {
       executeString("DROP TABLE \"" + getCurrentDatabase() + "\".\"TableName2\"");
     }
   }
+
+	@Test
+	public void testDump3() throws Exception {
+		if (!testingCluster.isHiveCatalogStoreRunning()) {
+			UserRoleInfo userInfo = UserRoleInfo.getCurrentUser();
+			ByteArrayOutputStream bos = new ByteArrayOutputStream();
+			PrintWriter printWriter = new PrintWriter(bos);
+
+			TajoDump.dump(client, userInfo, null, false, false, false, printWriter);
```

Comment:
It would be nice if the dump result is verified. Please refer to other tests in this class.